models/red_iris/red_iris.sdf

```diff
@@ -0,0 +1,544 @@
+<?xml version='1.0'?>
```

Comment:
@arwagoner can you find a way of simplifying this model where the only change is actually the color? First because it is the only change on the default iris model, second because we don't actually have an sdf for the iris model, since it is generated on build time.

Reply:
Yes, no problem. The original intent was to use this with multiple other iris models. I think the only reason I changed the color was to have a second iris model to use with a different mavlink port. Since the multi-uav issues have been worked out, those changes are no longer necessary.

Reply:
OK so can you update this PR so to just include the standard Iris model with the FPV camera? Thanks!